src/backend/optimizer/plan/plangroupext.c

```diff
@@ -430,11 +430,7 @@ plan_grouping_extension(PlannerInfo *root,
 	context.qual = qual;
 	context.numGroupCols = *p_numGroupCols;
 	context.grpColIdx = palloc0(context.numGroupCols * sizeof(AttrNumber));
-	memcpy(context.grpColIdx, *p_grpColIdx,
-		   context.numGroupCols * sizeof(AttrNumber));
 	context.grpOperators = palloc0(context.numGroupCols * sizeof(Oid));
-	memcpy(context.grpOperators, *p_grpOperators,
-		   context.numGroupCols * sizeof(Oid));
```

Comment:
context.grpColIdx and context.grpOperators are used [below](https://github.com/arenadata/gpdb/pull/733/files#diff-7191d843b4f61084e70f78d2a438eaa2eb8989b878ec38b4ca6067097f5dea85R486)